organization-understanding.md

```diff
@@ -0,0 +1,76 @@
+# Memorandum of Understanding
+
+**This is a draft.**
+
+[NumFOCUS](http://numfocus.org/)
+will apply to Google Summer of Code (GSoC)
+as a umbrella organization.
+[All projects/organizations supported by NumFOCUS](http://numfocus.org/projects/)
+can participate at GSoC under NumFOCUS umbrella.
+Projects/organizations not supported by NumFOCUS
+can participate at GSoC under NumFOCUS umbrella
+if approved by NumFOCUS board.
+
+## NumFOCUS Administrator for GSoC
+
+The NumFOCUS board will indicate one person,
```

Comment:
@moorepants I'm +1 for a committee and that's why I propose coordinators. The GSoC application need a name to be the administrator and that's why I suggest that NumFOCUS board indicate only one person.